src/windows-emulator/syscalls/token.cpp

```diff
@@ -236,7 +302,13 @@ namespace syscalls
 
         if (token_information_class == TokenIntegrityLevel)
         {
-            constexpr auto required_size = sizeof(sid) + sizeof(TOKEN_MANDATORY_LABEL64);
+            // NOLINTNEXTLINE(cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays)
+            const uint8_t medium_integrity_sid[] = {
```

Comment:
or should we allow c-style arrays? i honestly have no real preference and don't necessarily see the advantage of std::array here.

Reply:
I don't have a preference either tbh, but I saw code like this before when defining byte arrays in the code:
https://github.com/momo5502/sogen/blob/9e94c7d4cb37fa8dcc1b54246146103fca996c8e/src/windows-emulator/syscalls/token.cpp#L48-L52
https://github.com/momo5502/sogen/blob/9e94c7d4cb37fa8dcc1b54246146103fca996c8e/src/windows-emulator/apiset/default_apiset.hpp#L7-L10
https://github.com/momo5502/sogen/blob/9e94c7d4cb37fa8dcc1b54246146103fca996c8e/src/windows-emulator/apiset/default_apiset.hpp#L7-L10

So I thought keeping it consistent was a good idea.